contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -890,7 +890,7 @@ MakeEmptyParameterToken(char *name, int atttypid, int32 atttypmod, int attcollat
 				 * Get the precision and scale out of the typmod value if typmod is valid
 				 * Otherwise tds_default_numeric_precision/scale will be used.
 				 */
-				if (atttypmod >= VARHDRSZ)
```

Comment:
do we have test cases to cover all three case <, = and >?

Reply:
Yeah there are already tests which are testing this but I have confirmed with Satarupa that we should only check for `>` in above condition otherwise if atttypmod = VARHDRSZ, we will incorrectly return both precision and scale as zeros.

Reply:
is this only applicable to numeric zero? otherwise, we do not have test coverage for = condition which was broken so far

Reply:
Yeah atttypmod = VARHDRSZ only happens to be in case of numeric zero. I have added few more test cases for non zero numeric values.